addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -244,7 +257,7 @@ func on_item_used(item: PopochiuInventoryItem) -> void:
 ## Triggers the proper GUI command for the clicked mouse button identified with [param button_idx],
 ## which can be [enum MouseButton].MOUSE_BUTTON_LEFT, [enum MouseButton].MOUSE_BUTTON_RIGHT or
 ## [enum MouseButton].MOUSE_BUTTON_MIDDLE.
-func handle_command(button_idx: int) -> void:
+func handle_command(button_idx: int, double_click: bool = false) -> void:
```

Comment:
I suggest not changing the signature of this method since it's likely that the double click will only be considered when it's a left click (or a double tap on a touch screen). I'll explain myself in detail in the comment for line 357..